pkg/cmd/server/api/v1/types.go

```diff
@@ -143,6 +143,14 @@ type MasterConfig struct {
 	// omitted) and controller election can be disabled with -1.
 	ControllerLeaseTTL int `json:"controllerLeaseTTL"`
 
+	// AdmissionPluginOverride is a list of admission control plugins that will be installed
+	// on the Kubernetes master. Order is significant. If empty, a default list of plugins is used.
+	AdmissionPluginOverride []string `json:"admissionPluginOverride,omitempty"`
```

Comment:
Seems like we should pair these under a struct to keep them grouped together.  Also, didn't you convince me that we should have two separate chains?  One in `MasterConfig` and one in `KubernetesMasterConfig` because the code for registration will be separate until we properly tease apart API installer?